{{cookiecutter.project_shortname}}/Pipfile

```diff
@@ -7,9 +7,9 @@ verify_ssl = true
 check-manifest = ">=0.25"
 
 [packages]
-invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = "==0.6.0"}
-invenio-rdm-records = "==0.9.0"
-invenio-records-permissions = "==0.7.0"
+invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = "~=0.6.1"}
```

Comment:
Maybe is worth bumping this to 0.6.2 before merging. Tho it should work anyway.

Reply:
Sure. Updated it for appearances-sake but indeed it won't matter because the `~=` takes care of it.